.github/workflows/build.yaml

```diff
@@ -49,10 +49,12 @@ jobs:
 
       - name: Connectins snap interface
         run: |
-          sudo snap connect rt-tests:process-control :process-control
-          sudo snap connect rt-tests:mount-observe :mount-observe
-          sudo snap connect rt-tests:system-trace :system-trace
-          sudo snap connect rt-tests:cpu-latency-dev rt-tests:dma-latency-dev
+          sudo snap connect rt-tests:process-control
+          sudo snap connect rt-tests:mount-observe
+          sudo snap connect rt-tests:system-trace
+          sudo snap connect rt-tests:scheduler-debugfs
+          #sudo snap connect rt-tests:cpu-latency-dev rt-tests:dma-latency-dev
```

Comment:
Why is this commented out?

Reply:
It was before the fix-confinement branch has been synced to my current branch. I will put it back.